src/backend/dx11/src/factory.rs

```diff
@@ -129,20 +129,28 @@ impl Factory {
                               -> Result<h::RawBuffer<R>, buffer::CreationError> {
         use data::{map_bind, map_usage};
 
+        // we are not allowed to pass size=0, 
+        // otherwise it panics at self.share.handles.borrow_mut().make_buffer(buf, info, mapping)
+        let _size = if info.size == 0 {
```

Comment:
hmm, actually, can we please not call it `_size`? The underscore should be reserved for unused variables only.